src/treemap.ts

```diff
@@ -128,6 +134,8 @@ export class TreeMap {
     if (node.dom) return node.dom;
     const dom = document.createElement('div');
     dom.className = NODE_CSS_CLASS;
+    this.options.classList.forEach(nodeClass => dom.classList.add(nodeClass(node)));
+    this.setDOMColour(dom, this.options.nodeColor(node));
```

Comment:
I think maybe better to consistently use American spelling.  (This line has both colour and color in it.)

Reply:
Whoops, I meant to change all those to the American spelling, missed this one. Thanks for the catch.